packages/geocoder/src/geocoders/otp.ts

```diff
@@ -21,7 +21,23 @@ export default class OTPGeocoder extends Geocoder {
 
 
   rewriteAutocompleteResponse(response: OTPGeocoderResponse): MultiGeocoderResponse {
-    const generateLabel = stop => stop.code ? `${stop.name} (${stop.code})` : stop.name
+
+    const generateLabel = stop => {
+      let brackets = ""
+      if (stop?.agencies?.[0]?.name) {
+        brackets += stop.agencies[0].name
+      } else if (stop?.feedPublisher?.name) {
+        brackets += stop.feedPublisher.name
+      } 
+      if (stop?.code) {
+        if (brackets !== "") brackets += " "
+        brackets += stop.code
+      }
```

Comment:
This is probably fine, but it might be a bit cleaner depending on your opinion to make brackets an array, remove the logic around putting a space into it, and using array.join(" ") to convert it to a string.

Reply:
If we had more then 2 things in the brackets this would be good